tuna/mituna_interface.py

```diff
@@ -177,10 +190,11 @@ def get_kwargs(self, gpu_idx, f_vals):
         'job_queue_lock': f_vals["job_queue_lock"],
         'result_queue': f_vals["result_queue"],
         'result_queue_lock': f_vals["result_queue_lock"],
-        'label': self.args.label,
-        'docker_name': self.args.docker_name,
+        'label': self.args.label,  #type: ignore
+        'docker_name': self.args.docker_name,  #type: ignore
```

Comment:
why would you ignore the type here? they are both strings

Reply:
logic changed